### DISCUSSION TITLE
我的穿透流量某一天突然消失了一部分

### DISCUSSION THREAD

**Discussion by hite4044** [natfrp/wiki#298] [使用问题]
我是一个SakuraFrp的免费用户。
我在3月30号时穿透流量还剩104.3GB，我3月31号时发现流量变为61.4GB。
我查看了流量使用记录，发现3月30号那天只使用了6.8KB流量。
有谁解释一下为何我的流量消失了，谢谢！

**Comment by LTHPKBTE** [comment]
所有流量都有使用期限。请检查您的流量包过期情况。
https://www.natfrp.com/user/profile
